src/lib/encoder.ts

```diff
@@ -37,8 +37,12 @@ import {
   toChecksumAddress,
   utf8ToHex,
   stripHexPrefix,
+  hexToBytes,
+  bytesToHex,
 } from 'web3-utils';
 
+import BigNumber from 'bignumber.js';
```

Comment:
Let's not add this dep: fixed here: https://github.com/ERC725Alliance/erc725.js/pull/304